e2e-tests/conf/secrets.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: cluster1-secrets
+data:
```

Comment:
please use stringData as in https://github.com/percona/percona-xtradb-cluster-operator/blob/main/deploy/secrets.yaml

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/21/commits/8d6a0e91559771c3e2e21618eddc1c34fb559959